CCatHome_Regional_Host_Guide.md

```diff
@@ -0,0 +1,228 @@
+
+## Important Guidelines
+
+1. [Quick links and emergency contacts](https://docs.google.com/document/d/1VCDLt72aOwle9xTYzhbs4CHhnaZCvEzwLne5xb0QjLg/edit?usp=sharing)
+
+2. [Understanding your role as a host, co-host and CoC facilitator](https://carpentryconnect.org/online/#DefiningVolunteerRoles)
+
+3.  Useful links and guidelines for everyone:
+    - [Zoom Instructions](https://docs.google.com/document/d/1_7vxMtcqdn8s-zYOG6THnxZvs4uYmfJUySjNVeVFj98/edit?usp=sharing) in details
+
+    - Crowdcast Instruction (coomming soon)
+
+## Code of Conduct
+
+All hosts, speakers, and attendees of the CarpentryCon\@Home are
+Expected to adhere to the Carpentries Code of Conduct.
+
+- [Code of conduct: online documents](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html)
+
+- [Code of conduct shared folder](https://drive.google.com/drive/folders/1iQX5TFWuXmeXERgqR4zI0QaAulEj_6OD?usp=sharing)
+
+- Direct link to the [CoC Facilitators Roles and Responsibilities](https://docs.google.com/document/d/1p0GIrNzXHYJlJmFr_18oOkOIFRnfH51BqlBmqhNni-Y/edit?usp=sharing)
+
+## Role-Specific Guidelines for CarpentryCon@Home
+
+There are following roles in the CarpentryCon\@Home event beside the
+task force members:
+- Regional hosts
+- CoC facilitators who will be present as co-hosts with the regional hosts
+
+- Session lead: speakers, chairs, facilitators,
+
+### Instructions for the regional hosts
+
+
+#### General instructions 
+
+Detailed [Zoom Instructions](https://docs.google.com/document/d/1_7vxMtcqdn8s-zYOG6THnxZvs4uYmfJUySjNVeVFj98/edit?usp=sharing):
+-  Host Key: XXXX (Will be provided by The Carpentries staff in the taskforce)
+- To become the host (if you are using a Zoom room/account other than your own):
+
+    a.  Click on "Manage Participants" at the bottom on the screen.
+
+    b.  Click on "claim host" at the bottom of the Participants tab.
+
+    c.  Enter the host key
+
+- To assign the CoC facilitator and session lead a co-host in the call
+
+    a.  Open the "Participants" tab
+
+    b.  Find the names of these members
+
+    c.  Click "More"
+
+    d.  Select "Make Co-Host"
+
+- To record (if you don't have a Co-Host):
+
+    a.  Click on "Record" at the bottom of the screen.
+
+    b.  Then, click on "Record to the cloud".
+
+- To create breakout rooms:
+
+    a.  Click on "Breakout rooms" at the bottom of the screen.
+
+    b.  Select how many rooms to create.
+
+    c.  Click on "Create Breakout Rooms".
+
+    d.  Then, click on "Open All Rooms".
+
+    e.  You can send messages to all the groups via "Broadcast".
+
+#### Before the session
+
+-   Please read the [Zoom Instructions](https://docs.google.com/document/d/1_7vxMtcqdn8s-zYOG6THnxZvs4uYmfJUySjNVeVFj98/edit?usp=sharing) and test you Zoom account
+
+- Connect with your co-host/CoC facilitator and schedule a meeting with them to test the Zoom and explore the options you both will have available in Zoom
```

Comment:
There may not always be a co-host, so I would take this out.

Reply:
Agree. I still think that this part is important if there is a co-host. I will try to rephrase it. @malvikasharan what if there is no co-host? Is the regional coordinator going to be the CoC facilitator?

Reply:
@mkuzak We have now clarified that in the README for the regional hosts' role.

We should aim to select two regional host for each session, preferably one will be the main host and another a co-host in one session, and for another session they will switch roles.